### PR TITLE
Change to `development_board`

### PR DESCRIPTION
**Pull request by dunk2k**
## Description
changed keyboard to `development_board`, where `readme.md` or public accessible material(s) identify.  
Development boards in scope:
- blackpill_f401
- blackpill_f411
- bluepill
- proton_c

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* n/a

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by dunk2k** [comment]
> Not a good idea to perform this conversion blindly. The very first PCB modified in this submission (Acheron Project Shark, alpha version) cannot use the Proton-C dev board pinout [per its schematic](https://github.com/AcheronProject/SharkPCB/releases/download/Alpha/SharkPCB_schematic_Alpha.pdf).

appreciate the heads up @lesshonor 
On the subject of the `acheron/shark/alpha` - if it cannot use the Proton-C dev board, then to what purpose is `"board": "QMK_PROTON_C"` specified in `info.json`?

**Comment by sigprof** [comment]
`"board": "QMK_PROTON_C"` in many STM32F303-based keyboards probably comes from #10976.  Before that PR the default ChibiOS configuration for STM32F303 had most peripherals enabled by default; then #10976 introduced the `QMK_PROTON_C` board, which had most peripherals enabled, and switched the `GENERIC_STM32_F303XC` board to have all peripherals disabled; however, the same PR also added `BOARD = QMK_PROTON_C` to all previously existing STM32F303-based keyboards, so that the existing code would continue to work exactly as before.  Because of that, there are lots of keyboards which use the Proton C configuration, even though the actual hardware does not use the Proton C controller.

Switching the configuration to `"development_board": "proton_c"` for boards which don't actually use the Proton C controller still looks wrong (although it won't actually permit nonsensical conversions just yet, because the current converter support cannot handle conversion from Proton C to anything else — only `promicro` and `elite_c` controllers are supported as conversion sources).

**Comment by dunk2k** [comment]
Past history and all, reverting the proton_c development_board change until research denotes which boards are actual proton_c

**Comment by dunk2k** [comment]
Bump

**Review by drashna — APPROVED** [review]

**Comment by dunk2k** [comment]
Build failure reason:
- `handwired/splittest/bluepill:rgb`: missing ws2182 pin definition

**Review by fauxpark — APPROVED** [review]

File: keyboards/yanghu/unicorne/f411/info.json [hunk 1]
```diff
@@ -1,5 +1,3 @@
 {
```
Comment (zvecr) [inline keyboards/yanghu/unicorne/f411/info.json:1]: image

These need checking as i can see this doesnt use a development board, so this change is incorrect.

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/yanghu/unicorne/f411/info.json (hunk 1 above)
Comment (dunk2k) [reply keyboards/yanghu/unicorne/f411/info.json]: > Given this, I can only assume others are incorrect too.
re-evaluation done as per commit 7755f0e

**Review by fauxpark — APPROVED** [review]

**Comment by zvecr** [comment]
Loads of the changes are still wrong,

This is at least going to need to be partially reverted.

**Comment by fauxpark** [comment]
I went through all of them and they all seemed fine to me.

**Pull request merged**